product_supplierinfo_for_customer/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,3 @@
+For these prices to be used in sale prices calculations, you will have
+to create a pricelist with a rule with option "Based on" with the value
+"Partner Prices: Take the price from the customer info on the 'product form')".
```

Comment:
Maybe I'm just lost, but I don't see how you are supposed to create a pricelist with this option in later versions? in 14.0, 15.0 and 16.0 I can't visually see this field and in every view that it is included in it has `invisible=1`
I'm not sure why this is based on pricelists to begin with :thinking: since purchase orders don't work that way, but I suppose purchases don't have pricelists either to complicate it

Excuse the random ping @pedrobaeza but since you have contributed before maybe you have a better idea

Reply:
This is not for purchase orders, but sales orders, and the idea is to have a specific price for the pair customer/product without having to define a different pricelist per customer. You only set the pricelist rule "Partner Prices", and assign the same to all the customers, and then, price data is fetch from product.customerinfo table.